### DISCUSSION TITLE
MFCMAPI tool to fix The Folder Assistant cannot be displayed issue

### DISCUSSION THREAD

**Discussion by dpuffett** [microsoft/mfcmapi#569] [Q&A] [answered]
Having an issue with public folder - trying to modify an auto reply using the folder assistant - get an error The folder Assistant cannot be displayed. found this tool - however when I run it the cd: codes are different?

PR_EXTENEDED_RULE_MSG_CONDITION cb:113
PR_EXTENDED_RULE_MSG_ACTIONS cb:3

running our exchange in hybrid O365 - using outlook 2019 to manage PF folder assistant.

currently, this only effects this one public folder.

Thank you,
Daria

**Comment by stephenegriffin** [comment]
MFCMAPI is just showing the data it found. What is your question?

**Reply by dpuffett** [reply]
in the instruction step #8 in the properties pane should display cb:0 for the two Keys however cb: are not 0 can they be delete any way?

**Comment by dpuffett** [comment]
Stephen ,

Thank you for  taking the time to reply,

-  our issues is we have a PF that has an auto reply set – however, when we try to access the folder assistant to edit the auto reply template we received the error . the Folder Assistant cannot be displayed.

I did a google search and found the following
Delete corrupted public folder rules - Outlook | Microsoft Learn<https://learn.microsoft.com/en-us/outlook/troubleshoot/data-files/delete-corrupted-public-folder-rule>

In step # 8 - For each message that has the IPM.Rule.Version2.Message message class, check the properties in the pane beneath it. Find the messages for which the Properties pane displays cb: 0 in the Value column for the PR_EXTENDED_RULE_MSG_CONDITION and PR_EXTENDED_RULE_MSG_ACTIONS properties. Additionally, these properties might display PT_ERROR in the Type column and 0x0E9A000A in the Tag column. These properties indicate a corrupted rule.


I don’t have cb:0 in the value to those keys? See attached  - I was wondering if I could delete them anyway?

Thank you,
Daria Puffett

**Comment by WhiteZorin** [comment, marked as answer]
Hi Daria and rest of people looking for confirmation :)

After 3 years I found this topic as we had same issue with old Public Folders. IT WORKS :)

_I don’t have cb:0 in the value to those keys? See attached  - I was wondering if I could delete them anyway?_
- We also did not have cb:0, but all of 7 rules had different values
- Article talks about 4 places where you can find error values (none of them fitted for us)
- When we scrolled through the properties we found that many other different parameters have value "PT_ERROR"

We took the chance, deleted all the rules, closed MFCMAPI, opened Outlook and Folder Assistant started working instantaneously :)

Leaving it for the future, just in case someone ever finds it...
